### ISSUE TITLE
Doctest failures due to updates to astropy.stats.biweight_midvariance

### ISSUE THREAD

**Issue by bsipocz** [astropy/photutils#545]
Changes in https://github.com/astropy/astropy/pull/5991 broke some of the doctests in photutils. 
The examples should either be reworked or skipped until photutils requires astropy v2.0+

```
___________________ [doctest] docs/photutils/background.rst ____________________

081     5.2255295184

082     >>> print(biweight_location(data))

083     5.1867597555

084 

085 Similarly, using the biweight midvariance and median absolute

086 deviation to estimate the background noise level give values that are

087 larger than the true value of 2::

088 

089     >>> from astropy.stats import biweight_midvariance, mad_std

090     >>> print(biweight_midvariance(data))

Expected:

    2.22011175104

Got:

    4.96297528405

/tmp/photutils-test-ko6alo5x/docs/photutils/background.rst:90: DocTestFailure
```

**Comment by bsipocz** [comment]
Also I have noticed that the recent changes are not propagated over here to ``photutils.extern.stats``. 
I can open a PR for that if that was just an oversight, or do you prefer to track released versions only?

**Comment by larrybradley** [comment]
This was fixed in #547.
